FireRender.Maya.Src/FireRenderUtils.cpp

```diff
@@ -2263,3 +2263,13 @@ void GetUINameFrameExtPattern(std::wstring& nameOut, std::wstring& extOut)
 	extOut = nameOut.substr(lastDelimiter + 1, nameOut.length() - 1);
 	nameOut = nameOut.substr(0, firstDelimiter);
 }
+
+TimePoint GetCurrentChronoTime()
+{
+	return std::chrono::high_resolution_clock::now();
+}
+
+long TimeDiffChrono(TimePoint currTime, TimePoint startTime)
```

Comment:
Please consider making this function templatized, e.g. something like this:
```
template <typename T> 
long TimeDiffChrono(TimePoint currTime, TimePoint startTime)
{
	return (long)std::chrono::duration_cast<T>(currTime - startTime).count();
}

```
With use case like:
```
TimePoint x1, x2;
...
long res = TimeDiffChrono<std::chrono::seconds>(x1, x2);
```

So that we can avoid division by 1000

Reply:
Done